pymeasure/instruments/lecroy/lecroyT3DSO1204.py

```diff
@@ -1167,3 +1187,98 @@ def trigger(self):
         validator=strict_range,
         values=[-255, 255]
     )
+
+    ###############
+    #   Measure   #
+    ###############
+
+    _measurable_parameters = ["PKPK", "MAX", "MIN", "AMPL", "TOP", "BASE", "CMEAN", "MEAN", "RMS",
+                              "CRMS", "OVSN",
+                              "FPRE", "OVSP", "RPRE", "PER", "FREQ", "PWID", "NWID", "RISE", "FALL",
+                              "WID", "DUTY",
+                              "NDUTY", "ALL"]
+
+    measure_delay = Instrument.control(
```

Comment:
Watch out, you are triggering some linter reports - check the files tab, Github puts them there into "message bubbles".

Reply:
I think that I have solved them by now.